crates/nu-command/tests/commands/run_external.rs

```diff
@@ -213,14 +213,6 @@ fn external_command_not_expand_tilde_with_quotes() {
     )
 }
 
-#[test]
-fn external_command_receives_raw_binary_data() {
-    Playground::setup("external command receives raw binary data", |dirs, _| {
-        let actual = nu!(cwd: dirs.test(), pipeline(r#"0x[deadbeef] | table | bytes length"#));
-        assert_eq!(actual.out, r#"4"#);
-    })
-}
```

Comment:
Thanks for adding the testing utility!

Let's measure the `deadbeef` with your new testbin here :)

Reply:
@sholderbach makes sense. Didn't realize we could call testbins that way. I've added the test back as `0x[deadbeef] | nu --testbin input_bytes_length`

Reply:
Awesome thanks, all tests seem to go green!